README.md

```diff
@@ -1,56 +1,41 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Conference Page
```

Comment:
- [ ] Kindly update the title with the current project for a clear understanding of the topic.